seatunnel-connectors-v2/connector-tablestore/src/main/java/org/apache/seatunnel/connectors/seatunnel/tablestore/serialize/DefaultSeaTunnelRowSerializer.java

```diff
@@ -135,7 +139,8 @@ private Column convertColumn(String columnName, Object value, ColumnType columnT
             case BINARY:
                 return new Column(columnName, ColumnValue.fromBinary((byte[]) value));
             default:
-                throw new UnsupportedOperationException("Unsupported columnType: " + columnType);
+                throw new TablestoreConnectorException(TablestoreConnectorErrorCode.UNSUPPORTED_COLUMN_TYPE,
```

Comment:
CommonErrorCode.UNSUPPORTED_DATA_TYPE is better, please remove this redundant error code.